api/v2/db/ldap/connection.js

```diff
@@ -0,0 +1,28 @@
+const config = require('config');
+const ldap = require('ldapjs');
+
+const ldapConfig = config.get('dataSources').ldap;
+
+/**
+ * @summary Get an ldap connection
+ * @function
+ * @returns ldapjs client connection object
+ */
+const getClient = () => ldap.createClient({ url: ldapConfig.url });
```

Comment:
Looks like `ldapConfig.url` is the only field we need for creating the connection. You can use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) at line 4 to shorten the code:

```js
const { url } = config.get('dataSources.ldap');
```